hashicorp/boundary-ui
# Added Dropdown that matches with Hds

# Description
Updated Dropdown in the roles page to be configured alongside Hds

Currently, both dropdowns are on the page to show comparison between both


:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-13879)

## Screenshots (if appropriate)

## How to Test
1. Start up yarn server
2. Log in with dummy data
3. Select "Roles" in the dropdown
4. Select a role in the dropdown.
5. Manage dropdown menu should be on the top right. 'Delete Role' will remove the current role from the menu, and 'Add Principal' will add resource to current role. 

## Checklist


~- [] I have added before and after screenshots for UI changes~
~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
